src/util.h

```diff
@@ -326,6 +326,8 @@ inline bool StringEqualNoCase(const char* a, const char* b);
 // strncasecmp() is locale-sensitive.  Use StringEqualNoCaseN() instead.
 inline bool StringEqualNoCaseN(const char* a, const char* b, size_t length);
 
+inline uint32_t ReadUint32BE(const unsigned char* p);
```

Comment:
This should go into `base64.h` if the corresponding function is defined in `base64-inl.h`, I would say